### ISSUE TITLE
Home page resource cards heading accessibility

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#5119]
The Seeds homepage resources section has an improper order of heading priorities. Because the resources heading is an h2, the cards should be h3.

<img width="1746" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOFBQQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--45502dea6a14ce236de8164208246cfae1c8df14/Screenshot%202025-07-22%20at%208.21.11%E2%80%AFPM.png" alt="Screenshot 2025-07-22 at 8.21.11 PM.png" />
